packages/db/src/resources/compilations.ts

```diff
@@ -23,6 +23,21 @@ export const compilations: Definition<"compilations"> = {
       processedSources: [ProcessedSource]!
       sourceMaps: [SourceMap]
       contracts: [Contract]!
+      immutableReferences: [ImmutableReference]!
+    }
+
+    type ImmutableReference {
+      astNode: String!
+      bytecode: Bytecode!
```

Comment:
Just to be clear, that's a reference to a bytecode, right?  Like an ID or something?  I mean I have to assume so from context, but I don't know this code....

Reply:
Yes, correct!